halo/attest/keeper/keeper_internal_test.go

```diff
@@ -13,24 +13,196 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
-	fuzz "github.com/google/gofuzz"
 	"github.com/stretchr/testify/require"
 )
 
+// Hard-coded test data.
+//
+//nolint:gochecknoglobals // test data
+var (
+	blockHashes = []common.Hash{
```

Comment:
suggest using fuzzer or hex strings at least. This raw bytes is super verbose and doesn't mean anything to humans..